service/src/test/java/org/broadinstitute/listener/relay/http/AvailabilityTest.java

```diff
@@ -71,4 +87,44 @@ void livenessState() throws Exception {
         .andExpect(status().isServiceUnavailable())
         .andExpect(jsonPath("$.status").value("DOWN"));
   }
+
+  /**
+   * Verify that the HybridConnectionListenerHealth custom health check contributes to the UP/DOWN
+   * status of this app's liveness probe.
+   *
+   * @throws Exception on exception in the test
+   */
+  @Test
+  void hybridConnectionListenerHealth() throws Exception {
+    // livenessState and liveness probe should both be ok, to start
+    assertThat(
+        "Liveness state should be CORRECT",
+        applicationAvailability.getLivenessState(),
+        equalTo(LivenessState.CORRECT));
+    ResultActions result = mvc.perform(get("/actuator/health/liveness"));
```

Comment:
Is `result` leftover/redundant code?  It looks identical to `livenessResult` but with a less specific name.